### ISSUE TITLE
Default random name

### ISSUE THREAD

**Issue by wuniversales** [meshenger-app/meshenger-android#35]
It would be nice to have a random name by default, and then be able to change the name.

**Comment by mwarning** [comment]
You mean a  random name that is inserted in the initial setup window, so the user only need to click ok instead of inserting something?
Anyway, you can change the name afterwards in the settings menu.

**Comment by wuniversales** [comment]
Better not to ask anything, just to start the application with a random name, then if you want to configure the name manually, it can be changed in the settings

**Comment by mwarning** [comment]
in general, that is a good idea.

**Comment by iamvasuh** [comment]
The random name generated (only once) at the start should be different in each device.  @wuniversales please also specify, what will that random name consist of.

**Comment by ToEatFreshBabies** [comment]
consider random name to be English pronounceable instead of garbage block of consonants like fsdvgfdsnhmm

**Comment by wuniversales** [comment]
> The random name generated (only once) at the start should be different in each device. @wuniversales please also specify, what will that random name consist of.

@iamvasuh 
You could, for example, generate a hash based on the current time and date.
If you want the word to make sense, it would be more complicated ...

**Comment by eveofdestruction** [comment]
This seems more memorable @ToEatFreshBabies 

fire eat fish locker maven noble glitz moat

**Comment by mattcoxonline** [comment]
I have seen services before today, and I quite like this idea, that the username is constructed by using **adjective-randomword**

This generates usernames such as; _lame-bungalow_, _alarmed-horse_, _exclusive-winks_, _sugary-spider_. And a really simple way to do that would be to just have the placeholder text on this screen be generated from the word list: 

<img src="https://github.com/meshenger-app/meshenger-android/blob/master/docs/hello_2.0.0.png?raw=true" width="200" /> 

The "problem" here is that when you want to translate the app and support multiple languages then you have a lot of work on your hands. Or just don't suggest usernames for non-english languages.

**Comment by mwarning** [comment]
For now I would postpone this change as there are other pressing matters (that is a working/stable app).

**Comment by mwarning** [comment]
The Cancel button is now labelled Skip and then will generate a random name.
